front/app/containers/IdeasNewSurveyPage/IdeasNewSurveyForm/index.tsx

```diff
@@ -98,6 +99,9 @@ const IdeasNewSurveyForm = ({ project }: Props) => {
     : getCurrentPhase(phases?.data);
   const allowAnonymousPosting = phase?.attributes.allow_anonymous_participation;
 
+  // Used only to rerender the component when window is resized https://stackoverflow.com/a/38641993
```

Comment:
```suggestion
  // Used only to rerender the component when window is resized to recalculate the form's height https://stackoverflow.com/a/38641993
```
The comment misses an important piece: _why_ we are re-rendering. :)